tests/test-project/src/test/editor/textEditor.test.ts

```diff
@@ -323,14 +369,7 @@ describe('TextEditor', function () {
             await lens.click();
             await lens.getDriver().sleep(1000);
             const notifications = await new Workbench().getNotifications();
-            let notification: Notification;
-
-            for (const not of notifications) {
-                if ((await not.getMessage()).startsWith('CodeLens action clicked')) {
-                    notification = not;
-                    break;
-                }
-            }
+            const notification = notifications.find(async notification => { return (await notification.getMessage()).startsWith('Codelens action clicked'); });
```

Comment:
`find` does not support async callbacks; this will return a "truthy", unresolved Promise for each notification.

Reply:
good catch